tests/locustfile.py

```diff
@@ -0,0 +1,128 @@
+"""Run load tests for submitting folders and objects and querying them."""
```

Comment:
i think we can create a special folder just for these files called `performance` under `tests`